lib/commits.js

```diff
@@ -2,6 +2,34 @@ const _ = require('lodash')
 const { log } = require('./log')
 const { paginate } = require('./pagination')
 
+const findCommitsWithPathChangesQuery = ({ includePaths }) => /* GraphQL */ `
```

Comment:
I've done a bit of research into this and don't see a way to do this without inlining.
`history(path: String, ...` includes the following documentation:

> path ([String](https://docs.github.com/en/graphql/reference/scalars#string))
> 
> If non-null, filters history to only show commits touching files under this path.

which doesn't support an array. Attempting to unroll this and do the filter in JS would be possible, but instead of issuing an individual GraphQL query, we'd end up needing to round-trip to the API for each desired `path`.

What could be done is to transform the whole `... on Commit` in the main `findCommitsWithAssociatedPullRequests` into a fragment, parameterized by `path`, where if `includePaths` is an empty array we just pass `null`.

The downside there is that we'd then need to interpolate the resulting array of results ourselves, instead of what the current implementation does, selecting a group of node IDs that can be used to filter the array of returned PRs without altering the order.

I'm by far not an expert on either GraphQL itself, or GitHub's exposed GraphQL API, so I may have missed something and would be quite happy to revisit this approach, string-interpolating into a GraphQL query is undesirable from a number of perspectives, I just didn't see a way to do it without asking for support for this functionality from GitHub themselves.

Thanks for taking a look